src/ServiceInsight/MessageList/MessageListViewModel.cs

```diff
@@ -193,9 +192,9 @@ public void Handle(WorkFinished @event)
         public async Task Handle(SelectedExplorerItemChanged @event)
         {
             selectedExplorerItem = @event.SelectedExplorerItem;
-            selectedServiceControl = @event.SelectedExplorerItem.GetServiceControlClient(clientRegistry);
+            ServiceControl = @event.SelectedExplorerItem.GetServiceControlClient(clientRegistry);
```

Comment:
Was that a rule change? It's smart.

Reply:
Yeah, that was handled via the fixer. I was a bit mixed on keeping that change in since the property name and the previous field name weren't the same, but decided on to not suppress it.